integrated_channels/sap_success_factors/utils.py

```diff
@@ -77,6 +209,20 @@ def get_serialized_data(self):
     }
 
 
+def get_course_track_selection_url(enterprise_customer, course_id):
+    """
+    Given a course ID, and EnterpriseCustomer, craft a URL that links to the track selection page for that course.
+
+    Args:
+        enterprise_customer (EnterpriseCustomer): The EnterpriseCustomer that a URL needs to be built for
+        course_id (str): The string identifier of the course in question
+    """
+    netloc = enterprise_customer.site.domain
+    scheme = COURSE_URL_SCHEME
+    path = reverse('course_modes_choose', args=[course_id])
+    return urlunparse((scheme, netloc, path, None, None, None))
+
+
```

Comment:
Whoops, thought you were pointing to the other function! Will add a couple tests for this as well. :)